### DISCUSSION TITLE
★サーベル『グレイスワンダー』の隠し効果で幻覚耐性

### DISCUSSION THREAD

**Discussion by rurero2** [hengband/hengband#3737] [Ideas]
ほぼフレーバーですが、nethackより★サーベル『グレイスワンダー』に幻覚耐性を追加
幻覚付与時にチャージマン判定と同時にグレイスワンダー装備判定を行うという感じのを想定、nethackでは装備しているときだけ幻覚を見ない（外すと幻覚効果が再開する）というものでしたが、こちらの方が直感的で、面倒もなさそうなので…
[nethackwiki：グレイスワンダー](https://nethack-users.osdn.jp/hackaholic/index.php?%A5%B0%A5%EC%A5%A4%A5%B9%A5%EF%A5%F3%A5%C0%A1%BC)
[変愚wiki：グレイスワンダー](https://mars.kmc.gr.jp/~dis/heng_wiki/?%E3%82%A2%E3%83%BC%E3%83%86%E3%82%A3%E3%83%95%E3%82%A1%E3%82%AF%E3%83%88/%E3%82%B5%E3%83%BC%E3%83%99%E3%83%AB%E3%80%8E%E3%82%B0%E3%83%AC%E3%82%A4%E3%82%B9%E3%83%AF%E3%83%B3%E3%83%80%E3%83%BC%E3%80%8F)
[ビオ略：グレイスワンダー](https://beoline.nobody.jp/bandterm-ka.html#grayswandir) ビオ略によれば、原作で明確に幻覚耐性があるわけではない

**Comment by deskull-m** [comment]
採用の是非はともかく一度Issueに上げさせてもらいます。ありがとうございました。
